### ISSUE TITLE
Make errors

### ISSUE THREAD

**Issue by trans** [tonyrog/ffe#1]
Ran `make` and got this:

    ffe_beam.erl:32: function binary_to_atom/1 undefined
    ffe_beam.erl:45: function binary_to_atom/1 undefined
    ffe_beam.erl:64: function binary_to_atom/1 undefined
    ffe_beam.erl:11: Warning: export_all flag enabled - all functions will be exported
    make[1]: *** [Makefile:27: ../ebin/ffe_beam.beam] Error 1
    make[1]: Leaving directory '/home/trans/Workspace/ffe/src'

Never got to the C code. What is the C code for btw?

**Comment by tonyrog** [comment]
Erlang version? binary_to_atom has been around for a while now.
C code handle raw memory cells for constants, variables, arrays and also used while building code.
Please try again :-)

**Comment by trans** [comment]
You were right. Thank you. I had installed Elixir via asdf and Erlang came along with it. Not exactly sure what version it was using (think it was 21). But asdf hosed when I tried to upgrade to latest. So I had to wipe asdf from my system and use the official debian/ubuntu repository instead. That took care of this issue.

(Although another has now arrived, but I'll post that to a new ticket. And you close this one as resolved.)
